website-components-playground/src/components/playground-controller.tsx

```diff
@@ -55,18 +52,6 @@ const PlaygroundController = (props: TPlaygroundControllerProps) => {
               <Spacings.Stack scale="s">
                 <PreviewContainer height="400px">
                   <PortalsContainer />
-                  <Global
-                    // Overwrite styles to restrict the modal container to be within the
-                    // preview container instead of full screen.
-                    // This is only needed for the playground previews, therefore we
-                    // override the styles here.
-                    styles={css`
-                      .ReactModal__Body--open #${PORTALS_CONTAINER_ID} {
-                        position: absolute;
-                        overflow: auto;
-                      }
-                    `}
-                  />
```

Comment:
I’m removing this because I couldn’t figure out why it was needed and haven’t noticed any regressions.